source/tasks/Utils/install.ts

```diff
@@ -94,7 +94,10 @@ async function getOrDownloadOcto(option: DownloadOption, download?: (option: Dow
 }
 
 async function resolvePublishedOctoVersion(version?: string): Promise<DownloadOption> {
-    console.log(`Attempting to contact ${OctopurlsUrl} to find latest CLI tools`);
+    if (version === null || version === undefined) {
```

Comment:
Can this be 
```
if (version) {
```
That should check for both `undefined` and `null`

Reply:
Okay. Changed to ```if (!version)```